web/src/components/molecules/Common/MultiValueField/MultiValueAsset/index.tsx

```diff
@@ -0,0 +1,140 @@
+  onAssetsCreate: (files: UploadFile[]) => Promise<(Asset | undefined)[]>;
+  onAssetCreateFromUrl: (url: string) => Promise<Asset | undefined>;
+  onAssetsReload: () => void;
+  onAssetSearchTerm: (term?: string | undefined) => void;
+  setFileList: (fileList: UploadFile<File>[]) => void;
+  setUploadModalVisibility: (visible: boolean) => void;
+};
+
+const MultiValueAsset: React.FC<Props> = ({
+  className,
+  value = [],
+  onChange,
+  assetList,
+  fileList,
+  loadingAssets,
+  uploading,
+  uploadModalVisibility,
+  uploadUrl,
+  uploadType,
+  onUploadModalCancel,
+  setUploadUrl,
+  setUploadType,
+  onAssetsCreate,
+  onAssetCreateFromUrl,
+  onAssetsReload,
+  onAssetSearchTerm,
+  setFileList,
+  setUploadModalVisibility,
+  ...props
+}) => {
+  const t = useT();
+  const handleInput = (e: string, id: number) => {
+    onChange?.(value?.map((valueItem, index) => (index === id ? e : valueItem)));
+  };
+
+  const deleteInput = (key: number) => {
+    onChange?.(
+      value.filter((_, index) => {
+        return index !== key;
+      }),
+    );
+  };
+
+  return (
+    <div className={className}>
+      {Array.isArray(value) &&
+        value?.map((valueItem, key) => (
+          <FieldWrapper key={key}>
+            <FieldButton
+              type="link"
+              icon={<Icon icon="arrowUp" />}
+              onClick={() => onChange?.(moveItemInArray(value, key, key - 1))}
+              disabled={key === 0}
+            />
+            <FieldButton
+              type="link"
+              icon={<Icon icon="arrowDown" />}
+              onClick={() => onChange?.(moveItemInArray(value, key, key + 1))}
+              disabled={key === value.length - 1}
+            />
+            <AssetItem
+              {...props}
+              value={valueItem}
+              assetList={assetList}
+              fileList={fileList}
+              loadingAssets={loadingAssets}
+              uploading={uploading}
+              uploadModalVisibility={uploadModalVisibility}
+              uploadUrl={uploadUrl}
+              uploadType={uploadType}
+              onUploadModalCancel={onUploadModalCancel}
+              setUploadUrl={setUploadUrl}
+              setUploadType={setUploadType}
+              onAssetsCreate={onAssetsCreate}
+              onAssetCreateFromUrl={onAssetCreateFromUrl}
+              onAssetsReload={onAssetsReload}
+              onAssetSearchTerm={onAssetSearchTerm}
+              setFileList={setFileList}
+              setUploadModalVisibility={setUploadModalVisibility}
+              onChange={(e: string) => handleInput(e, key)}
+            />
+            <FieldButton
+              type="link"
+              icon={<Icon icon="delete" />}
+              onClick={() => deleteInput(key)}
+            />
+          </FieldWrapper>
+        ))}
+      <Button
+        icon={<Icon icon="plus" />}
+        type="primary"
+        onClick={() => {
+          if (!value) value = [];
```

Comment:
Again, you are already setting a default value so you don't need this

Reply:
`null` is a value that won't trigger the default value to be used in this case.